torch/testing/_internal/common_utils.py

```diff
@@ -1517,7 +1517,11 @@ def allocator_option_enabled_fn(allocator_config, _, option):
     # other libraries take up about a little under 1 GB of space per process
     torch.cuda.set_per_process_memory_fraction(round((gb_available - num_procs * .85) / gb_available / num_procs, 2))
 
+
+GPU_TYPE = "xpu" if torch.xpu.is_available() else "cuda"
 requires_cuda = unittest.skipUnless(torch.cuda.is_available(), "Requires CUDA")
+requires_gpu = unittest.skipUnless(torch.cuda.is_available()
```

Comment:
Different with the `requires_gpu` in `torch/testing/_internal/inductor_utils.py` here we don't need to check if has triton.